contactform.php

```diff
@@ -158,15 +349,23 @@ public function getTemplateVarOrders()
         return $orders;
     }
 
+    /**
+     * @throws PrestaShopDatabaseException
+     * @throws PrestaShopException
+     */
     public function sendMessage()
     {
         $extension = array('.txt', '.rtf', '.doc', '.docx', '.pdf', '.zip', '.png', '.jpeg', '.gif', '.jpg');
         $file_attachment = Tools::fileAttachment('fileUpload');
         $message = Tools::getValue('message');
+        $url = Tools::getValue('url');
+        $clientToken = Tools::getValue('token');
+        $serverToken = $this->context->cookie->contactFormToken;
+        $clientTokenTTL = $this->context->cookie->contactFormTokenTTL;
 
         if (!($from = trim(Tools::getValue('from'))) || !Validate::isEmail($from)) {
             $this->context->controller->errors[] = $this->trans('Invalid email address.', array(), 'Shop.Notifications.Error');
-        } elseif (preg_match('#^[ ]*$#', $message)) {
+        } elseif (!$message) {
```

Comment:
It seems a message filled with blanks could pass now, should not we test that with a `trim()` and `empty()`

Reply:
go for empty 👍